Assets/MRTK.ThirdParty.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Should this stay in the ThirdParty folder, given that it's being brought in a bit more to the MRTK? Keeping it here would also interfere with anyone who imports the original extension on top of MRTK 2.5.

Reply:
I think people who import the original extension will end up having to deal with issues since we still rely on the underlying OculusQuestHand class. Keeping it in a ThirdParty folder I think better helps highlight the idea that this module was largely built upon one of our community members, though I guess it still doesn't fix the issue that MRTK-Quest users will face in trying to import that library.

Reply:
For now going to move it under Assets/Providers/Oculus/MRTK.ThirdParty to alight with our name space adjustments

Reply:
Also saw your earlier comment about namespaces, so it should work out fine 👍